changelog/fragments/default-helm-reconcile-period.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      Changed the default reconcile period of Helm-based operators to 10h instead of 1m.
+    kind: "change"
```

Comment:
Switching this to the default value as specified by controller-runtime's `SyncPeriod` makes sense. However, since this is a user-facing change, I think this needs a doc on "why we made it to 10h specifically". Having some reference on this as mentioned [here](https://github.com/kubernetes-sigs/controller-runtime/issues/521#issuecomment-518457302) would be helpful imo, or else it may be weird to users on why we suddenly modified the default to "10h" specifically.